Added check for relative links to non-markdown files (#2559)
openvinotoolkit/open_model_zoo: data/datasets.md

        * `images` - directory with converted images
        * `depth` -  directory with depth maps

> Note: If dataset is used in the first time, please set `allow_convert_data: True` in annotation conversion parameters for this dataset in `dataset_definitions.yml`  or use `<omz_dir>/tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py` and  following command line to get converted data .

Reviewer: ```suggestion
Note: If dataset is used in the first time, please set `allow_convert_data: True` in annotation conversion parameters for this dataset in `dataset_definitions.yml`  or use `convert_annotation` command line interface:
```
